gruntfileConfig.json

```diff
@@ -4,6 +4,7 @@
   "src": "./src",
   "assets": "./src/assets",
   "theme": "./node_modules/patternpack-example-theme",
+  "logo": "/theme-assets/images/pp-logo.svg",
```

Comment:
I'm not sure I follow your logic. There are two options:

1. Don't put a value in for `options.logo`. This will default to the logo that's provided by `patternpack-example-theme` which is located in the URL above.
2. Pass in a value for `options.logo`, which should be a path to your logo file. If you're using the default theme, you wouldn't have the ability to put anything in the `theme-assets` folder from the example theme, so you *could* put it anywhere, but `assets/images/` seems like a logical place.

This option is not needed if you have a custom `options.theme` value because then you'd be managing your own page template.